composer.json

```diff
@@ -59,7 +59,7 @@
         "optimize-autoloader": true,
         "sort-packages": true,
         "platform": {
-            "php": "7.2.28"
+            "php": "7.2"
```

Comment:
Past versions of Composer have interpreted "7.2" as "7.2.0". If this hasn't changed, using this limit would prevent dependencies requiring e.g. ^7.2.5 from being used. Unless there is a specific reason why the shorter version is now better, I'd prefer to specify the patch version here.

Reply:
The change was left accidentally. I wanted to check symfony 4.